gensrc/thrift/PlanNodes.thrift

```diff
@@ -318,6 +318,7 @@ struct TOlapScanNode {
   23: optional map<i32, i32> dict_string_id_to_int_ids
   // which columns only be used to filter data in the stage of scan data
   24: optional list<string> unused_output_column_name
+  25: optional bool is_lake_table
```

Comment:
Is it appropriate to add a new `TLakeScanNode`, instead of reuse the `TOlapScanNode`?

Reply:
done